recipes/dd-agent.rb

```diff
@@ -44,6 +44,16 @@
   include_recipe 'datadog::_install-linux'
 end
 
+
+if node['datadog']['agent_enable'] && !node['datadog']['enable_process_agent']
```

Comment:
Shouldn't it be the other way around? We want to log if the Agent is **not** enabled and the Process Agent **is** enabled. Maybe I am reading it incorrectly 🤔 
```suggestion
if !node['datadog']['agent_enable'] && node['datadog']['enable_process_agent']
```

Reply:
My mistake, you are reading it right.